### ISSUE TITLE
Enum dropdown not showing when using patternProperties for schema

### ISSUE THREAD

**Issue by ziga-miklic** [josdejong/jsoneditor#1111]
Hello,

I have the following schema:

```js
{
  title: 'Ports',
  description: 'Object containing ports',
  type: 'object',
  patternProperties: {
    '.*': {
      type: 'object',
      properties: {
        testrigId: {
          enum: this.devicePorts,
        },
      },
      required: [
        'testrigId',
      ],
    }
  },
};
```

The problem is that the `testrigId` property is not showing a dropdown, but a string input. A similar schema using `properties` instead of `patternProperties` is displaying the dropdown.

![issue](https://user-images.githubusercontent.com/3201207/93480738-0f484e80-f8fe-11ea-839f-7dfd3e47f555.png)
![working](https://user-images.githubusercontent.com/3201207/93480742-0fe0e500-f8fe-11ea-993d-353fe1545af8.png)

I see that this was fixed in #486, but I also see someone having the same issue this year in #904 but the issue was only closed.

Is this an issue with JSON Editor or am I doing something wrong?

Thank you!

**Comment by josdejong** [comment]
Thanks for reporting Žiga. I just tried it out and I can reproduce this issue with the schema above, there is indeed no enum being displayed.

Anyone able to look into this issue? Help would be very welcome.

**Comment by ziga-miklic** [comment]
Hello, @josdejong thank you for your response. I can give it a go and try to fix it.

I'll let you know how it goes.

**Comment by josdejong** [comment]
Thanks 👍

**Comment by josdejong** [comment]
Fix (#1116) has been published in `v9.1.1`, thanks again!
